seirmo/model.py

```diff
@@ -0,0 +1,37 @@
+#
+# ForwardModel Class
+#
+
+
+class ForwardModel:
+    """ForwardModel Class:
+
+    Abstract base class for the SEIR model class.
```

Comment:
This isn't actually the base class for the SEIR model specifically, but for any model in the seir repository (this could in theory be other models too, such as the SIR, or SMIR model.